user/cowsql/template.py

```diff
@@ -0,0 +1,23 @@
+pkgname = "cowsql"
+pkgver = "1.15.6"
+pkgrel = 0
+build_style = "gnu_configure"
+configure_gen = ["autoreconf", "-i"]
+hostmakedepends = [
+    "automake",
+    "slibtool",
+    "bash",
+    "pkgconf",
```

Comment:
sort

Reply:
this goes for the other *depends lists too when they're vertical

Reply:
done